include/revng/Support/Debug.h

```diff
@@ -282,3 +282,6 @@ extern Logger<> NRALog;
 extern Logger<> PassesLog;
 extern Logger<> ReleaseLog;
 extern Logger<> VerifyLog;
+
+void writeToFile(const llvm::StringRef What,
+                 const llvm::StringRef Path) debug_function;
```

Comment:
Passing `const` by value does not make a lot of sense.
Also, keep in mind that `StringRef` and `ArrayRef` already represent read-only references (you can't make changes to the pointed data).

Reply:
oops, fixed removing const.